CHANGES.md

```diff
@@ -1,6 +1,18 @@
 Change Log
 ==========
 
+### 1.42 - 2018-03-01
+
+* Added a post-processing framework. [#5615](https://github.com/AnalyticalGraphicsInc/cesium/pull/5615)
+   * Added `Scene.postProcessStages` which is a collection of post-process stages to be run in order.
+      * Has a built-in `ambientOcclusion` property which will apply screen space ambient occlusion to the scene and run before all stages.
+      * Has a built-in `bloom` property which applies a bloom filter to the scene before all other stages but after the ambient occlusion stage.
+      * Has a built-in `fxaa` property which applies Fast Approximate Anti-aliasing (FXAA) to the scene after all other stages.
+   * Added `PostProcessStageLibrary` which contains several built-in stages that can be added to the collection.
+   * Added `PostProcessStage` which takes a fragment shader that processes the color and depth texture from the stage run before it.
```

Comment:
At one point we use to document the `czm_` built-in GLSL functions, but we stopped.

Is there a meaningful subset of them they are widely used when writing post-processing stages?  If so, we should consider adding them back to the reference doc.

Reply:
There are a few, like the constants, RGB conversions, and the sun position. There are a few that convert the screen space coordinates to eye coordinates. Would we want to document one's we think are useful or all of them?

Reply:
Let's just document the ones we need, but it is a good beginner issue we can do later, see #6185